src/Xrefcheck/Core.hs

```diff
@@ -180,25 +180,25 @@ pattern PathSep <- (isPathSeparator -> True)
 
 -- | Type of reference.
 data LocationType
-  = LocalLoc
-    -- ^ Reference on this file
+  = CurrentFileLoc
+    -- ^ Reference on this file, e.g. [a](#header)
```

Comment:
Please surround the examples with `@`, like `@[a](#header)`. This will format them as inline code, like the backtick syntax in markdown.